main/src/main/java/org/mobilitydata/gtfsvalidator/notice/RouteSameNameAndDescriptionNotice.java

```diff
@@ -45,6 +45,6 @@ public SameNameAndDescriptionForRouteNotice(
 
   @Override
   public String getCode() {
-    return "same_route_name_and_description";
+    return "route_same_name_and_description_notice";
```

Comment:
The _notice suffix is redundant.

Reply:
Removed in https://github.com/MobilityData/gtfs-validator/pull/792/commits/7296922fb39a2ce31b79a67345180d082422c57a. Thanks!